scripts/configure-hypervisor.sh

```diff
@@ -317,12 +317,7 @@ generate_qemu_options() {
 	qemu_options+=(size:--disable-tools)
 
 	# Disable XEN driver
-	case "$arch" in
-	aarch64) ;;
-	x86_64) qemu_options+=(size:--disable-xen) ;;
-	ppc64le) qemu_options+=(size:--disable-xen) ;;
-	s390x) qemu_options+=(size:--disable-xen) ;;
-	esac
+	qemu_options+=(size:--disable-xen)
```

Comment:
thanks, could you please update the commit message? :smile_cat:

Reply:
sure， updated